fe-nc-news-react/src/components/FullArticle.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { fetchArticlebyId as fetchArticlesId } from '../api';
+import CircularProgress from "@mui/material/CircularProgress";
+import Box from "@mui/material/Box";
+
+export default function FullArticle({ isLoading, setIsLoading}) {
+    const [article, setArticle] = useState({})
+
+    const { article_id } = useParams()
+
+    useEffect(() => { 
+        setIsLoading(true);
+        fetchArticlesId(article_id).then((res) => {
+            setArticle(res.data.article)
+        })
+            .then(() => {
+                setIsLoading(false)
+            }).catch((err) => {
+                setIsLoading(false)
+            });
+    }, [article_id])
+    return isLoading ? (
+      <Box sx={{ display: "flex", justifyContent: "center" }}>
+        <CircularProgress />
+      </Box>
+    ) : (
+      <div>
+        <h2>{article.title}</h2>
+        <ul>
+          <li>
+            <img src={article.article_img_url} alt="article image" />
+            <p>Date Written: {article.created_at}</p>
```

Comment:
consider this for displaying information: https://www.refactoringui.com/previews/labels-are-a-last-resort